_includes/pbc/all/install-features/202403.0/install-glue-api/install-the-company-account-glue-api.md

```diff
@@ -0,0 +1,917 @@
+
+
+{% info_block errorBox %}
+
+The following feature integration guide expects the basic feature to be in place. It adds only the Company Account REST API functionality.
+
+{% endinfo_block %}
+
+This document describes how to install the Company Account feature API.
+
+## Prerequisites
+
+To start the feature integration, overview and install the necessary features:
+
+| FEATURE OR GLUE API  | VERSION    | INSTALLATION GUIDE    |
+| ----------------- | ---------- | --------------------- |
+| Glue API: Spryker Core                | {{page.version}} | [Install the Spryker Core Glue API](/docs/pbc/all/miscellaneous/{{page.version}}/install-and-upgrade/install-glue-api/install-the-spryker-core-glue-api.html) |
+| Company Account                       | {{page.version}}  | [Company account feature integration](/docs/pbc/all/customer-relationship-management/{{page.version}}/base-shop/install-and-upgrade/install-features/install-the-company-account-feature.html) |
+| Glue API: Customer Account Management | {{page.version}} | [Glue API: Customer Account Management feature integration](/docs/pbc/all/identity-access-management/{{page.version}}/install-and-upgrade/install-the-customer-account-management-glue-api.html) |
+| Glue API: Glue Application            | {{page.version}} | [Glue API: Glue Application feature integration](/docs/pbc/all/miscellaneous/{{page.version}}/install-and-upgrade/install-glue-api/install-the-spryker-core-glue-api.html) |
+| UUID Generation Console               | {{page.version}}  |  |
```

Comment:
* We have to add a link to the document.
* The public link is the following: https://docs.spryker.com/docs/pbc/all/cart-and-checkout/202311.0/base-shop/install-and-upgrade/install-features/install-the-uuid-generation-console-feature.html#prerequisiteshttps://docs.spryker.com/docs/pbc/all/cart-and-checkout/202311.0/base-shop/install-and-upgrade/install-features/install-the-uuid-generation-console-feature.html#prerequisites